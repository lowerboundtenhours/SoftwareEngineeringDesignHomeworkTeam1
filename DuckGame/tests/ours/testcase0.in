duck MallardDuck CanFly Quack
duck RedheadDuck CanFly Quack
duck RubberDuck CannotFly Squeak
duck DecoyDuck CanotFly Silent
swim MallardDuck
swim RedheadDuck
swim RubberDuck
swim DecoyDuck
fly MallardDuck
fly RedheadDuck
fly RubberDuck
fly DecoyDuck
quack MallardDuck
quack RedheadDuck
quack RubberDuck
quack DecoyDuck