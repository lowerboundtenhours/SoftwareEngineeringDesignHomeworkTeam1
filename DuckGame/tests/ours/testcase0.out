MallardDuck is swimming
RedheadDuck is swimming
RubberDuck is swimming
DecoyDuck is swimming
duck is flying
duck is flying
duck cannot fly
duck cannot fly
quack quack quack
quack quack quack
squeak squeak squeak
(silent)